apps/frontend/src/components/global/Footer.vue

```diff
@@ -3,6 +3,17 @@
     <span class="bar-visible--text">
       The MITRE Corporation &copy; 2018-{{ new Date().getFullYear() }}
     </span>
+    <v-spacer />
+    <a href="https://www.netlify.com">
+      <img
+        :src="require('@/assets/netlify-light.svg')"
+        style="width: 1.5rem; height: 1.5rem; vertical-align: middle"
+        class="mr-4"
+      />
+    </a>
+    <a href="https://github.com/mitre/mitre-saf" style="text-decoration: none">
```

Comment:
this should point to the heimdall2 repo right?

Reply:
Yes, I had previously fixed that, but I had undone some changes so I must have gone back too far.  Thank you for brining that to my attention.

Reply:
Yes, I had fixed that, but I had undone something else so I must have gone back too far.  Thank you for bringing it to my attention.

Reply:
cool